### ISSUE TITLE
"Remember me" option

### ISSUE THREAD

**Issue by Abyss777** [vitalidze/traccar-web#174]
It's given:
I have to enter username and password in **desktop** browser only once until server reboot.
I have to reenter username and password almost all times after i've reconnected from 3g to wifi and back (and even while moving along the city) in **mobile** phone.

I suppose web session connected to client IP.
Is it possible to disconnect web session from client IP or add tick to logon screen like "Remember Me" ?

**Comment by vitalidze** [comment]
Actually it does exact same thing for authentication as the desktop version. It stores a special Cookie with session identification, which has `Sesssion` expiration, which means that it expires once you close the web browser. But yes, I think this can be updated by using a cookie with hard coded expiration date. Moreover, some time ago I thought about storing session information directly in Cookie so it will survive even server restarts.

**Comment by Abyss777** [comment]
Hm. It's a feature of desktop Chrome. It doesn't clear "session" cookies on exit if it's configured as "Continue where you left off."
But mobile Chrome sometimes looses session even if it is not closed. (i.e. I'm driving a car and watching device moving in mobile phone through 3G. At some point, device stops moving on map, I'm refreshing the page and have to enter credentials again.)
May be mobile Chrome clear "session" cookies on loosing internet connection...(it's hard to debug)

Anyway, It would be nice to have two features:
- Tick "Remember me" on logon screen
- Button in Users dialog "Forget user sessions" if user selected in list and "Forget all users sessions" if no one user selected. Or your version how to implement this. This feature is for security reasons.

**Comment by crarchile** [comment]
+1
